01-main/packages/freeplane

```diff
@@ -0,0 +1,11 @@
+DEFVER=1
+get_website "https://sourceforge.net/projects/freeplane/files/freeplane%20stable/"
+if [ "${ACTION}" != "prettylist" ]; then
+    # Find the latest .deb URL for the current naming pattern
+    local REDIR_URL="$(grep -Eo 'https://downloads\.sourceforge\.net/project/freeplane/[^"\ >]+' "${CACHE_FILE}" | grep 'freeplane_[0-9.]*~upstream-[0-9]+_all.deb' | head -n1)"
```

Comment:
This currently fails for me and needs a tweak. Also we can use just one `grep` and remove an unnecessary `head` . 

```suggestion
    local REDIR_URL=$(grep -m 1 -Eo  'https://sourceforge.net/projects/freeplane/files/freeplane%20stable/freeplane_[1-9].*\.deb/download' $CACHE_FILE )
```